test/index.test.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const assert = require('assert');
+const JsonML = require('..');
+const dom = require('../lib/dom');
+const html = require('../lib/html');
+const utils = require('../lib/utils');
+
+describe('JsonML module index', function() {
+  it('should merge and export interfaces', function() {
+    assert.deepEqual(JsonML, Object.assign({}, dom, html, utils));
```

Comment:
👌

Reply:
I don't love that it's essentially repeating the source but i thought maybe it was an ok shorthand? I duplicated the effort below to be more exacting

Reply:
this is ok with me 👍  

alternatively, we could create lists of the keys exported by the `dom`, `html`, and `utils` modules.
we would then combine those lists here and assert against the merged list

i feel that the current approach still gets the idea across :)